src/controller_examples/rabbitmq_controller/proof/liveness/helper_invariants/invariants.rs

```diff
@@ -242,40 +370,31 @@ pub proof fn lemma_true_leads_to_always_at_most_one_create_cm_req_is_in_flight(s
                                 }
                             );
                             seq_lemmas::seq_equal_preserved_by_add_prefix(key.name, new_strlit("-server-conf")@, new_strlit("-plugins-conf")@);
-                            if s.message_in_flight(msg) {
-                                assert(input.0.is_Some());
-                                assert(resp_msg_matches_req_msg(input.0.get_Some_0(), s.pending_req_of(key)));
-                                assert(false);
-                            } else {
-                                assert(at_rabbitmq_step(key, RabbitmqReconcileStep::AfterCreateServerConfigMap)(s_prime));
-                            }
+                            assert(!s.message_in_flight(msg));
+                            assert(at_rabbitmq_step(key, RabbitmqReconcileStep::AfterCreateServerConfigMap)(s_prime));
                         }
                     },
-                    Step::ClientStep(input) => {
-                        assert(s.message_in_flight(msg));
+                    _ => {
+                        assert(requirements(msg, s));
                         assert(s.reconcile_state_of(key) == s_prime.reconcile_state_of(key));
-                        assert(at_rabbitmq_step(key, RabbitmqReconcileStep::AfterCreateServerConfigMap)(s_prime));
-                        assert(s_prime.network_state.in_flight.count(msg) == 1);
-                    },
-                    _ => {}
+                    }
                 }
             }
         }
     }
-    invariant_action_n!(
-        spec, stronger_next, RMQCluster::every_new_req_msg_if_in_flight_then_satisfies(requirements),
+    invariant_n!(
```

Comment:
It seems that from the documentation of `invariant_n!`, it requires that `partial_spec == p1 /\ p2 /\ ... /\ pn` -- where did you prove that equality in this lemma?

Reply:
oh I see, you don't mean `==` but actually `.eq`. And `==` is proved by `combine_spec_entails_always_n`. Could you update the documentation for both `combine_spec_entails_always_n` and `invariant_n` to make it clear?